query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/discover/LdapTestBind.java

```diff
@@ -74,6 +81,25 @@ public BooleanField performFunction() {
         return new LdapTestBind();
     }
 
+    @Override
+    public void validate() {
+        super.validate();
+
+        if (containsErrorMsgs()) {
+            return;
+        }
+
+        if (DIGEST_MD5_SASL.equals(creds.bindMethod())) {
+            // To use MD5, we require an encrypted connection
+            if (!(START_TLS.equals(conn.encryptionMethod()) //
+                    || LDAPS.equals(conn.encryptionMethod()))) {
+                ReportImpl report = new ReportImpl();
+                report.addArgumentMessage(new ErrorMessageImpl(MD5_NEEDS_ENCRYPTED));
```

Comment:
Any reason we don't make this into a static method getter with all the others in LdapMessages? It's nice having everything centralized there.